data/transport missions.txt

```diff
@@ -39,6 +39,36 @@ mission "Immigrant Workers"
 				accept
 
 
+mission "Vintage Spaceman"
+	description "A young man wants to find love on <destination>."
+	minor
+	passengers 1
+	to offer
+		random < 50
+	source "X"
+	destination
+		attributes core
+	on complete
+		payment 10000
+		conversion
+			`	"We're here!" Vintage spaceman slams the emergency unlock and jumps out while your ship is still a meter off the ground. He runs off in to the crowd.`
+			`	You grab a drink, climb on top of the <ship> and relax.`
+			`	After the concert is over you see your passenger emerge from the crowd, his head hung low. "It's all a ruse man. She was just an bot. A marketing tool to get me to buy tickets."`
+			`	"Guess she *was* made for you." You quip, feeling a little bad about it.`
+			`	"Sorry I wasted your time. I'm such a sucker. I don't have any credits for you, but you can have this suit. I don't feel like wearing it any more."`
+			`	He leaves the suit on your deck and disappears back in to the crowd. You take the suit to a pawn shop and get X credits for it.`
```

Comment:
> "We're here!" The vintage spaceman slams the emergency unlock and jumps out while your ship is still a meter off the ground. He runs off into the crowd.

> You grab a drink, climb on top of the <ship> and relax.

> After the concert is over, you see your passenger emerge from the crowd; his head is hung low. "It's all a ruse man," he says. "She was just a bot. A marketing tool to get me to buy tickets."

> "Guess she `*was*` made for you." You quip, feeling a little bad about it.

Unsure about what to say here. I assume that the `*was*` part represents italicized text, and even if it doesn't, I've never seen it before in Michael's works...
> "Sorry I wasted your time," he says. "I'm such a sucker. I don't have any credits for you, but you can have this suit. I don't feel like wearing it any more."

> He leaves the suit on your deck and disappears back into the crowd. You take the suit to a pawn shop and receive <payment> for it.